src/components/create-guild/Requirements/SetRequirements.tsx

```diff
@@ -91,9 +93,19 @@ const SetRequirements = ({ titleSize = undefined }: Props): JSX.Element => {
                     type={type}
                     field={field as Requirement}
                     index={i}
-                    removeRequirement={removeReq}
+                    removeRequirement={(idx) => {
+                      if (type === "GUILD_SNAPSHOT") {
```

Comment:
Right now you cannot add a snapshot requirement just by itself, it only gets created if you make a token reward; so I thought this could be a simple and quick assumption, that there should be a token reward there always. 
The most accurate solution would be to check if there is a token reward that has the requirement's id in its dynamic value params, which, as of now is a bit complicated.
Maybe we could make some hooks that enable checking these connections easier

Reply:
Yup, the hook is a great idea! Then let's add a comment here to explain why it is sufficient to check only for the type here 😉